infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -19,6 +19,9 @@ project_id = "k8s-infra-oci-proxy-prod"
 // gcr.io/k8s-staging-infra-tools/archeio:v20230310-v0.2.0@sha256:bc742c5f47a69e21e828768991853faddbe13a7f69a9da4d7d2ad16e0e55892c
 // If you're interested in running this image yourself releases are available at registry.k8s.io/infra-tools/archeio
 digest = "sha256:bc742c5f47a69e21e828768991853faddbe13a7f69a9da4d7d2ad16e0e55892c"
+// we increase this in staging, but not in production
+// we already get a lot of info from build-in cloud run logs
+verbosity = "0"
```

Comment:
TODO: consider v=1 for default?

v=0 should match klog default, aiming for no status quo change with this PR, just a variable that _could_ override it